app/models/match.rb

```diff
@@ -10,6 +10,9 @@ class Match < ApplicationRecord
   validates :receiver_id, presence: true
   validates :received, inclusion: { in: [true, false] }
 
+  scope :received, -> { where(received: true) }
+  scope :not_received, -> { where(received: false) }
```

Comment:
If it's for search purposes wouldn't they do `received:false`?

Reply:
That would be better, but I'm using a fork that does it this way. I believe we should not bother to re-implement that this way because it would be a great exercise to redo this with a dedicated frontend.

Basically, you define scopes and you search with them by using `<name of scope>:`, notice the colon at the end.

Reply:
Wait, they don't support values? So you need to create a new scope for the negation of a scope?

Reply:
Yes, this is what we must live with atm.

Reply:
¯\\\_(ツ)\_/¯ I don't agree with it but it's what we have to work with. Let's take it forward!

Reply:
@justmendes in the summer we'll make this great again with a proper admin dashboard